pootle/apps/pootle_app/views/admin/projects.py

```diff
@@ -30,11 +30,16 @@ def get_context_data(self, **kwargs):
         except Language.DoesNotExist:
             default_language = languages[0].id
 
+        filetypes = []
+        for info in formats.get().values():
+            filetypes.append(
+                [info["pk"], "%s(%s)" % (info["title"], info["extension"])])
```

Comment:
No space between the title and the parentheses? I wonder if this string could be provided by the format itself.